docs/reference/ml/apis/evaluateresources.asciidoc

```diff
@@ -0,0 +1,50 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-evaluate-dfanalytics-resources]]
+=== Evaluate {dfanalytics} resources
+
+The evaluation object defines the type of evaluation you want to perform. An 
+evaluation configuration object has the following properties:
+
+`binary_soft_classification`::
```

Comment:
I think here we should list the `evaluation` field and explain this can be a different object depending on the type of evaluation. Then we can link to the section below where we explain each evaluation type (even though at the moment there is only one: `binary_soft_classification`).